scene/2d/skeleton_2d.cpp

```diff
@@ -325,7 +325,7 @@ bool Bone2D::_editor_get_bone_shape(Vector<Vector2> *p_shape, Vector<Vector2> *p
 		rel = rel.rotated(-get_global_rotation()); // Undo Bone2D node's rotation so its drawn correctly regardless of the node's rotation
 	} else {
 		real_t angle_to_use = get_rotation() + bone_angle;
-		rel = Vector2(cos(angle_to_use), sin(angle_to_use)) * (length * MIN(get_global_scale().x, get_global_scale().y));
+		rel = Vector2(cos(angle_to_use), sin(angle_to_use)) * length * get_global_scale().abs();
```

Comment:
This PR is attempting to give scale to `length` and could probably be solved by changing how `length` is calculated: https://github.com/godotengine/godot/pull/83397#discussion_r1426076290